baselines/teams.md

```diff
@@ -88,114 +87,141 @@ need.
 
 3.  Select the **Global (Org-wide default)** policy.
 
-4.  Under the **Content sharing** section, set **Allow an external
-    participant to give or request control** to **Off**.
+4.  Under the **Content sharing** section, set **External
+    participants can give or request control** to **Off**.
 
 5.  If custom policies have been created, repeat these steps for each
     policy, selecting the appropriate policy in step 3.
 
-## 2. Starting Teams Meetings
-
-This setting controls which meeting participants can start a meeting. In
-this instance, the term “anonymous users” refers to any Teams users
-joining calls that are not authenticated through the agency’s tenant.
+## 2. Automatic Admittance into meetings
+This setting controls which meeting participants wait in the lobby
+before they are admitted to the meeting.
 
 ### Policies
 
 #### MS.TEAMS.2.1v1
-Anonymous users SHALL NOT be enabled to start meetings in the Global (Org-wide default) meeting policy or in custom meeting policies if any exist.
+Anonymous users, including dial-in users, SHOULD NOT be admitted automatically in the Global (Org-wide default) meeting policy.
 - _Rationale:_ TODO add rationale.
 - _Last modified:_ July 2023
+- Custom meeting policies MAY be created that allow more flexibility for specific users.
+
+#### MS.TEAMS.2.2v1
+Internal users SHOULD be admitted automatically in the Global (Org-wide default) meeting policy.
+- _Rationale:_ TODO add rationale.
+- _Last modified:_ July 2023
+- Custom meeting policies MAY be created that allow more flexibility for specific users.
+
+#### MS.TEAMS.2.3v1
+B2B guest users MAY be admitted automatically in the Global (Org-wide default) meeting policy.
+- _Rationale:_ TODO add rationale.
+- _Last modified:_ July 2023
+- Custom meeting policies MAY be created that allow more flexibility for specific users.
+
+
+
 
 ### Resources
 - [Meeting policy settings - Participants & guests \| Microsoft
-Docs](https://docs.microsoft.com/en-us/microsoftteams/meeting-policies-participants-and-guests)
+  Docs](https://docs.microsoft.com/en-us/microsoftteams/meeting-policies-participants-and-guests)
 
 ### License Requirements
 - N/A
 
 ### Implementation
-To configure settings for anonymous users:
+All the settings in this section are configured in the [**Microsoft Teams admin
+    center**](https://admin.teams.microsoft.com).
+
+#### MS.TEAMS.2.1v1, instructions:
 
 1.  Sign in to the [**Microsoft Teams admin
     center**](https://admin.teams.microsoft.com).
+2.  Select **Meetings >** **Meeting** **policies**.
+
+3.  Select the **Global (Org-wide default)** policy.
 
-2.  Select **Meetings -\>** **Meeting policies**.
+4.  Under the **Meeting join & lobby** section **,** ensure
+    **Who can bypass the lobby** is *not* set to **Everyone**. Bypassing the lobby should be set to **People in my org**, though other options may be used if needed. 
+
+5.  In the same section, set **People dialing in can bypass the lobby** to
```

Comment:
I think the configuration for "People dialing in can bypass the lobby SHOULD be set to off" should be represented in the policy description because it is a separate config in the Teams admin center. Instead of trying to add it to the existing policy, maybe just create a new one in this same section. It is currently missing from the policies. We had a similar case in AAD and I ended up creating a policy for it during Emerald.

![image](https://github.com/cisagov/ScubaGear/assets/107076927/97557781-7e7b-4212-a458-cdee1249d682)

Reply:
Policy 2.1 includes dialing in users:
![MS_TEAMS_2 1](https://github.com/cisagov/ScubaGear/assets/104993331/80607174-9c34-41cc-b789-a3f76b88e540)